Examples/RPC/go/README.md

```diff
@@ -0,0 +1,15 @@
+# Go HTTP RPC Sample
+
+This directory contains a sample Go program that calls the Google Natural Language API using HTTP, RPC, and Protocol Buffers.
```

Comment:
The phrase "HTTP, RPC, and protocol buffers" is weird to me. HTTP is a transport, protocol buffers is a wire format, and RPC is a....something. Technique?

If I approach this from the perspective of an unfamiliar face, I think I would be likely to think, "Okay, I know what HTTP is, but what is RPC and what are protocol buffers?" We can (and probably should) provide a link to the protobuf home page on each of these for the latter. For RPC, I think we either need something explaining what we mean here, or we should just remove it from _this_ list, even if we do want to explain the RPC technique elsewhere.

Reply:
Rephrasing this to "This directory contains a sample Go program that calls the Google Natural Language API using the gRPC Fallback protocol described in HowToRPC."